### PR TITLE
feat: support Vite v6 in mocker package

### PR DESCRIPTION
**Pull request by cexbrayat**
### Description

This avoids warnings like:

```
└─┬ vitest 3.0.0-beta.2
  └─┬ @vitest/mocker 3.0.0-beta.2
    └── ✕ unmet peer vite@^5.0.0: found 6.0.3
```

when using v3 beta with Vite v6 and pnpm.

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. If the feature is substantial or introduces breaking changes without a discussion, PR might be closed.
- [ ] Ideally, include a test that fails without this PR but passes with it.
- [ ] Please, don't make changes to `pnpm-lock.yaml` unless you introduce a new test example.

### Tests
- [x] Run the tests with `pnpm test:ci`.

### Documentation
- [ ] If you introduce new functionality, document it. You can run documentation with `pnpm run docs` command.

### Changesets
- [ ] Changes in changelog are generated from PR name. Please, make sure that it explains your changes in an understandable manner. Please, prefix changeset messages with `feat:`, `fix:`, `perf:`, `docs:`, or `chore:`.

### CODE REVIEW

File: packages/ui/package.json [hunk 1]
```diff
@@ -81,7 +81,7 @@
     "unocss": "^0.65.1",
     "unplugin-auto-import": "^0.18.6",
     "unplugin-vue-components": "^0.27.5",
-    "vite": "^5.0.0",
+    "vite": "^5.0.0 || ^6.0.0",
```
Comment (sheremet-va) [inline packages/ui/package.json:84]: this is a dev dep, it shouldn't be updated

Comment (cexbrayat) [reply packages/ui/package.json]: Oh right, my bad. I updated the PR 👍

**Review by sheremet-va — APPROVED** [review]

**Comment by sheremet-va** [comment]
Thank you!

**Comment by dubzzz** [comment]
Is there a plan to release this change within a 2.x version?

**Comment by sheremet-va** [comment]
> Is there a plan to release this change within a 2.x version? (if vite 6.x is compatible with 2.x)

Vitest 2 will not support Vite 6, that’s the whole reason we are releasing Vitest 3

**Pull request merged**